util/types/etc.go

```diff
@@ -62,6 +62,12 @@ func IsTypeFractionable(tp byte) bool {
 	return tp == mysql.TypeDatetime || tp == mysql.TypeDuration || tp == mysql.TypeTimestamp
 }
 
+// IsTypeTime returns a boolean indicating
+// whether the tp is time type like datetime, date or timestamp.
+func IsTypeTime(tp byte) bool {
```

Comment:
There are many other IsTypeXXX functions in this file, 
I'm not sure whether it'll be called in other packages,
so I put it here.